polarion_rest_api_client/open_api_client/models/workitems_list_get_response_data_item_relationships_approvals_data_item_type.py

```diff
@@ -7,7 +7,7 @@
 class WorkitemsListGetResponseDataItemRelationshipsApprovalsDataItemType(
     str, Enum
 ):
-    WORKRECORDS = "workrecords"
+    WORKITEM_APPROVALS = "workitem_approvals"
 
```

Comment:
Should be
```
WORKRECORDS = "workrecords"
WORKITEM_APPROVALS = "workitem_approvals"
```

Reply:
Why? This is the enumeration of the relationships approvals data item type, which should not be workrecords. There is also [this enum](https://github.com/DSD-DBS/polarion-rest-api-client/blob/update-auto-generated-client/polarion_rest_api_client/open_api_client/models/workitems_list_get_response_data_item_relationships_work_records_data_item_type.py)